src/cc/frontends/clang/b_frontend_action.cc

```diff
@@ -1154,8 +1154,11 @@ bool BTypeVisitor::VisitCallExpr(CallExpr *Call) {
             error(GET_BEGINLOC(Call), "invalid bpf_table operation %0") << memb_name;
             return false;
           }
-          prefix += "((void *)bpf_pseudo_fd(1, " + fd + "), ";
-
+          if (memb_name == "call") {
+            prefix += "(bpf_pseudo_fd(1, " + fd + "), ";
+          } else {
+            prefix += "((void *)bpf_pseudo_fd(1, " + fd + "), ";
+          }
```

Comment:
This looks inconsistent with other helpers. Instead, I think we should change the signature of `bpf_tail_call_`:
https://github.com/iovisor/bcc/blob/f279656f431aba104cd0ae12ff9af3aac3175e5b/src/cc/export/helpers.h#L552-L554

to:
```c
void bpf_tail_call_(void *map_fd, void *ctx, int index) { 
   ((void (*)(void *, u64, int))BPF_FUNC_tail_call)(ctx, map_fd, index); 
 } 
```

Reply:
Done